firebase-sessions/src/main/kotlin/com/google/firebase/sessions/SessionFirelogPublisher.kt

```diff
@@ -35,21 +35,24 @@ internal class SessionCoordinator(
       try {
         firebaseInstallations.id.await()
       } catch (ex: Exception) {
-        Log.e(TAG, "Error getting Firebase Installation ID: ${ex}. Using an empty ID")
+        Log.e(tag, "Error getting Firebase Installation ID: ${ex}. Using an empty ID")
         // Use an empty fid if there is any failure.
         ""
       }
 
     try {
       eventGDTLogger.log(sessionEvent)
 
-      Log.i(TAG, "Successfully logged Session Start event: ${sessionEvent.sessionData.sessionId}")
+      Log.i(tag, "Successfully logged Session Start event: ${sessionEvent.sessionData.sessionId}")
     } catch (ex: RuntimeException) {
-      Log.e(TAG, "Error logging Session Start event to DataTransport: ", ex)
+      Log.e(tag, "Error logging Session Start event to DataTransport: ", ex)
     }
   }
+  val tag = "SessionFirelogPublisher"
```

Comment:
Just out of curiosity, why have this here instead of as a const in the companion object?

Reply:
yeah, I don't know. Felt like a const in a companion object was heavy, but I suppose I can stick it there.